receiver/webhookeventreceiver/metadata.yaml

```diff
@@ -8,3 +8,7 @@ status:
   codeowners:
     active: ["atoulme", "shalper2"]
     emeritus:
+
+tests:
+  config:
+    endpoint: 127.0.0.1:8088
```

Comment:
could this be anyport instead of a specific one?

Reply:
It has to be a valid port, 0 won’t work. Note the test does not set up a server on that port.